docs_versioned_docs/version-ros2humble/ros/installation/upgrading.mdx

```diff
@@ -102,10 +105,576 @@ The backup script will copy the following data:
 - `apt` packages
 - User Permission Groups
 
-#### Backing up non-standard data
+### Backing up non-standard data
 
-The `backup.sh` script assumes that your robot is in a roughly-standard configuration; it uses a single user account and no files within `/opt/ros/melodic`have been modified.
+The `backup.sh` script assumes that your robot is in a roughly-standard configuration; it uses a single user account and no files within `/opt/ros/$ROS_DISTRO`have been modified.
 
 If this is not the case, it is the responsibility of the user to ensure that any modified files and files in other users’ home folders is backed up correctly.
 
 A common example of this might be if you have created customized URDF files to be loaded via environment variables (e.g. `JACKAL_URDF_EXTRAS`) and have stored them outside any of the folders specified above, you must back these up yourself; `backup.sh` will not do this for you.
+
+## Converting Environment Variables to Clearpath Config
+
+The environment variables used to define your robot in ROS 1 and ROS 2 Foxy can be converted to the `robot.yaml` format to maintain the same robot description when you upgrade to ROS 2 Humble.
+Below are some common examples. Visit the [robot.yaml documentation](../config/yaml.mdx) for more details.
+
+### Attachments
+
+Each platform has platform-specific attachments (formerly decorations) such as bumpers and top plates. These now reside under the [platform](../config/yaml.mdx#platform) section of the `robot.yaml` file.
+
+<Tabs groupId="platform_codes">
+  <TabItem
+    value="A200"
+    label="A200"
+  >
+
+Environment variables:
+
+```
+export HUSKY_FRONT_BUMPER=1
+export HUSKY_REAR_BUMPER=1
+export HUSKY_FRONT_BUMPER_EXTEND=0
+export HUSKY_REAR_BUMPER_EXTEND=0.1
+export HUSKY_WIBOTIC_BUMPER=1
+export HUSKY_TOP_PLATE_ENABLED=1
+export HUSKY_LARGE_TOP_PLATE=1
+export HUSKY_USER_RAIL_ENABLED=0
+export HUSKY_SENSOR_ARCH=1
+export HUSKY_SENSOR_ARCH_HEIGHT='510'
+export HUSKY_SENSOR_ARCH_OFFSET='0 0 0'
+export HUSKY_SENSOR_ARCH_RPY='0 0 0'
+```
+
+`robot.yaml` configuration:
+
+```
+platform:
```

Comment:
I would like to see an even more straight forward connection between the environment variables and the config. Some times the data isn't presented in the exact same way as in the envars so I think it might cause for incorrect assumptions and support requests. Can we add comments to the config excerpts to make it more clear? An example of what I am thinking is shown below:

```
platform:
  attachments:
    front_bumper:
      enabled: true                         # HUSKY_FRONT_BUMPER
      model: wibotic                        # HUSKY_WIBOTIC_BUMPER (1 => model: wibotic, 0 => model: default)
      xyz: [0.0, 0.0, 0.0]
      rpy: [0.0, 0.0, 0.0]
      extension: 0.0                        # HUSKY_FRONT_BUMPER_EXTEND
    rear_bumper:
      enabled: true                         # HUSKY_REAR_BUMPER
      model: default
      xyz: [0.0, 0.0, 0.0]
      rpy: [0.0, 0.0, 0.0]
      extension: 0.1                        # HUSKY_REAR_BUMPER_EXTEND
    top_plate:
      enabled: true                         # HUSKY_TOP_PLATE_ENABLED
      model: large                          # HUSKY_LARGE_TOP_PLATE (1 => model: large, 0 => model: pacs)
      xyz: [0.0, 0.0, 0.0]
      rpy: [0.0, 0.0, 0.0]
    structure:
      enabled: true                         # HUSKY_USER_RAIL_ENABLED? or HUSKY_SENSOR_ARCH?
      model: sensor_arch_510                # HUSKY_SENSOR_ARCH_HEIGHT (510 => sensor_arch_510, 300 => sensor_arch_300)
      xyz: [0.0, 0.0, 0.0]                  # HUSKY_SENSOR_ARCH_OFFSET
      rpy: [0.0, 0.0, 0.0]                  # HUSKY_SENSOR_ARCH_RPY
```